### PR TITLE
Misc tweaks to `tools/perf.sh`

### PR DESCRIPTION
**Pull request by bfops**
# Description of Changes
Made a few minor updates to `tools/perf.sh`:
- explicitly check & log "found multiple candidate processes" separately from "no process found"
- only grabs processes that look like `spacetime start`
    - previously this would complain about e.g. `spacetime logs`
- show spacetime processes from **all** users on the machine

# API and ABI breaking changes
None

# Expected complexity level and risk
2

*How complicated do you think these changes are? Grade on a scale from 1 to 5,
where 1 is a trivial change, and 5 is a deep-reaching and complex change.*

*This complexity rating applies not only to the complexity apparent in the diff,
but also to its interactions with existing and future code.*

*If you answered more than a 2, explain what is complex about the PR,
and what other components it interacts with in potentially concerning ways.*

### CODE REVIEW

File: tools/perf.sh [hunk 1]
```diff
@@ -85,12 +85,18 @@ if [[ -z "$DATAFILE" ]]; then
     DATAFILE=perf.data
 fi
 if [[ -z "$SPACETIME_PID" ]]; then
-    if [[ $(ps -a | grep spacetime | wc -l) != 1 ]] ; then
+    SPACETIMES="$(ps -a -e -f | grep '\<spacetime\>.*\<start\>' | grep -v '\<grep\>')"
```
Comment (bfops) [inline tools/perf.sh:88]: `-f` allows us to get the args
`-e` lets us see all users

Comment (gefjon) [reply tools/perf.sh:88]: This would make a great comment to add to the code.

**Review by gefjon — APPROVED** [review]
Can you go through and change all the error-path `echo`s to `>&2 echo`? Other than that, this looks good!

File: tools/perf.sh (hunk 1 above)
Comment (cloutiertyler) [reply tools/perf.sh:88]: Although, that's a bigger change

**Pull request merged**